CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 ## CHANGE LOG
 
+### 1.9.0
```

Comment:
Does this warrant a new major version?

Reply:
I was debating that. I don't think so since the update is for the codebase and not any features. Do you think it should be a major bump?